src/Auth_Command.php

```diff
@@ -91,6 +109,50 @@ public function create( $args, $assoc_args ) {
 		}
 	}
 
+	/**
+	 * Helper function for `ee auth create admin-tools`
+	 * Creates auth for `admin-tools`
+	 *
+	 * @param array $assoc_argsassoc arguments passed to ee auth create.
+	 *
+	 * @return void
+	 */
+	private function admin_tools_create_auth( $assoc_args ) {
+		verify_htpasswd_is_present();
+
+		$user = EE\Utils\get_flag_value( $assoc_args, 'user' );
+
+		if ( ! $user ) {
```

Comment:
Please do allow random usernames for the sake of consistency

Reply:
`ee auth create admin-tools` supports random usernames.